### PR TITLE
Removed usage of deprecated v1 Bitstamp API

### PR DESCRIPTION
**Pull request by tommq**
I had some issues with BTC withdrawals and ended up doing quite some changes including:

- Removing the deprecated v1 API entirely (https://www.bitstamp.net/api-deprecated/)
- Adding functionality to withdraw multiple new coins (XLM, USDC, LINK and more)

Fixes for:
- Cancel all orders: expected boolean to be returned when in reality it is JSON
- XRP deposit address: was returning Ripple IOU address instead of XRP. There are now two separate methods.

### CODE REVIEW

**Comment by tommq** [comment]
According to emails set out by Bitstamp the v1 API will stop working on the 1.12.2021. Can someone take a look at this PR?

**Review by jtrimmer-nydig — APPROVED** [review]
Looks good to me. Hope this can get in before the old Bitstamp API goes dead.

File: xchange-bitstamp/src/main/java/org/knowm/xchange/bitstamp/service/BitstampAccountServiceRaw.java [hunk 1]
```diff
@@ -14,556 +9,591 @@
...
-  }
+    /**
+     * This method can withdraw any currency if withdrawal endpoint is configured in BitstampAuthenticatedV2
+     **/
+    public BitstampWithdrawal withdrawBitstampFunds(
+            Currency currency, BigDecimal amount, final String address, final String tag)
+            throws IOException {
+        BitstampWithdrawal response;
+
+        amount = amount.setScale(5, RoundingMode.HALF_UP);
```
Comment (jtrimmer-nydig) [inline xchange-bitstamp/src/main/java/org/knowm/xchange/bitstamp/service/BitstampAccountServiceRaw.java:96]: Why was this rounding to 5 decimal places added? Is there some significant figure requirement with the new API? I am not seeing this in the new Bitstamp API documentation?

**Review by jtrimmer-nydig — APPROVED** [review]

File: xchange-bitstamp/src/main/java/org/knowm/xchange/bitstamp/service/BitstampAccountServiceRaw.java (hunk 1 above)
Comment (jtrimmer-nydig) [reply xchange-bitstamp/src/main/java/org/knowm/xchange/bitstamp/service/BitstampAccountServiceRaw.java]: Super, thanks! Tested with BTC too.

**Pull request merged**